packages/cactus-plugin-keychain-memory/src/main/typescript/get-prometheus-exporter-metrics/get-prometheus-exporter-metrics-endpoint-v1.ts

```diff
@@ -0,0 +1,81 @@
+import { Express, Request, Response } from "express";
+
+import {
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+  Checks,
+} from "@hyperledger/cactus-common";
+
+import {
+  IWebServiceEndpoint,
+  IExpressRequestHandler,
+} from "@hyperledger/cactus-core-api/";
+
+import OAS from "../../json/openapi.json";
+
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+
+import { PluginKeychainMemory } from "../plugin-keychain-memory";
+
+export interface IGetPrometheusExporterMetricsEndpointV1Options {
+  logLevel?: LogLevelDesc;
+  connector: PluginKeychainMemory;
+}
+
+export class GetPrometheusExporterMetricsEndpointV1
+  implements IWebServiceEndpoint {
+  private readonly log: Logger;
+
+  constructor(
+    public readonly opts: IGetPrometheusExporterMetricsEndpointV1Options,
+  ) {
+    const fnTag = "GetPrometheusExporterMetricsEndpointV1#constructor()";
+
+    Checks.truthy(opts, `${fnTag} options`);
+    Checks.truthy(opts.connector, `${fnTag} options.connector`);
+
+    this.log = LoggerProvider.getOrCreate({
+      label: "get-prometheus-exporter-metrics-v1",
+      level: opts.logLevel || "INFO",
+    });
+  }
+
+  public getExpressRequestHandler(): IExpressRequestHandler {
+    return this.handleRequest.bind(this);
+  }
+
+  public getPath(): string {
+    return OAS.paths[
+      "/api/v1/plugins/@hyperledger/cactus-plugin-keychain-memory/get-prometheus-exporter-metrics"
+    ].get["x-hyperledger-cactus"].http.path;
+  }
+
+  public getVerbLowerCase(): string {
+    return OAS.paths[
+      "/api/v1/plugins/@hyperledger/cactus-plugin-keychain-memory/get-prometheus-exporter-metrics"
+    ].get["x-hyperledger-cactus"].http.verbLowerCase;
+  }
+
+  public registerExpress(app: Express): IWebServiceEndpoint {
+    registerWebServiceEndpoint(app, this);
+    return this;
+  }
+
+  async handleRequest(req: Request, res: Response): Promise<void> {
+    const fnTag = "GetPrometheusExporterMetrics#handleRequest()";
+    const verbUpper = this.getVerbLowerCase().toUpperCase();
+    this.log.debug(`${verbUpper} ${this.getPath()}`);
+
+    try {
+      const resBody = await this.opts.connector.getPrometheusExporterMetrics();
```

Comment:
@jagpreetsinghsasan I'd call it something else, not a connector. Either would call it maybe `keychain` or just go with `plugin` which makes it more copy-paste-proof for the future ;-)